dotnet/src/Microsoft.Agents.AI/ChatClient/ChatClientAgent.cs

```diff
@@ -322,7 +322,6 @@ public override async IAsyncEnumerable<AgentRunResponseUpdate> RunStreamingAsync
     public override AgentThread GetNewThread()
         => new ChatClientAgentThread
         {
-            MessageStore = this._agentOptions?.ChatMessageStoreFactory?.Invoke(new() { SerializedState = default, JsonSerializerOptions = null }),
```

Comment:
Just deleting this is sufficient because we already lazy-initialize it to the exact same thing later?